third_party/suitesparse.BUILD

```diff
@@ -300,6 +312,8 @@ cc_library(
         "CHOLMOD/SuiteSparse_metis/include/",
     ],
     linkopts = [
+        "-Lexternal/aarch64-sysroot/usr/lib/aarch64-linux-gnu/lapack",
```

Comment:
I thought about this a little more and we're probably going to need to guard these in a `select` statement so that they're only put on the link line when we're actually using the sysroot.

The way bazel's build tree is structured - the `external/aarch64-sysroot/` folder may be on the path even when not doing a build against a sysroot. This could cause the build to attempt to link libs for the wrong platform.

Reply:
I've added the appropriate guards.

Reply:
Thanks.